docker/prod/reactjs/nginx.conf

```diff
@@ -0,0 +1,36 @@
+# ide_nginx.conf
+
+# the upstream component nginx needs to connect to
+upstream django {
+    server 0.0.0.0:8000; # for a web port socket (we'll use this first)
+}
+
+# configuration of the server
+server {
+    # the port your site will be served on
+    listen      80;
+    # the domain name it will serve for
+    server_name 0.0.0.0; # substitute your machine's IP address or FQDN
+    charset     utf-8;
+
+    # max upload size
+    client_max_body_size 75M;   # adjust to taste
+
+    # Django media
+    location /media  {
+        alias /code/media;  # your Django project's media files - amend as required
+    }
+
+    location /static {
+        alias /code/static; # your Django project's static files - amend as required
+    }
+
+    # Finally, send all non-media requests to the Django server.
+    location / {
+        uwsgi_pass  django;
+        uwsgi_read_timeout 1800;
+        proxy_read_timeout 300;
+    	  proxy_connect_timeout 300;
+    	  proxy_pass http://0.0.0.0:8000;
```

Comment:
Replace this with `proxy_pass django`

Reply:
I feel that there is something wrong in the nginx configurations. Can you please check EvalAI Nginx configuration and if there is something wrong in this configuration?

Reply:
same nginx conf works on my local system